spec/lib/billy/cache_spec.rb

```diff
@@ -78,7 +78,7 @@
       end
 
       it 'More specifically, the cache keys should be identical for the 2 analytics urls' do
-        identical_cache_key = 'post_5fcb7a450e4cd54dcffcb526212757ee0ca9dc17'
+        identical_cache_key = 'post_www.example-analytics.com_5fcb7a450e4cd54dcffcb526212757ee0ca9dc17'
```

Comment:
@maukoquiroga, since this changed, I suspect these changes will require a caching reset to regenerate the cache?